public/app/features/teams/TeamSettings.tsx

```diff
@@ -21,13 +24,28 @@ export type Props = ConnectedProps<typeof connector> & OwnProps;
 
 export const TeamSettings: FC<Props> = ({ team, updateTeam }) => {
   const canWriteTeamSettings = contextSrv.hasPermissionInMetadata(AccessControlAction.ActionTeamsWrite, team);
+  const currentOrgId = contextSrv.user.orgId;
+
+  const [{ roleOptions }, setOrgId] = useRoleOptions(currentOrgId);
+  const [pendingRoles, setPendingRoles] = useState<Role[]>([]);
+
+  const canUpdateRoles =
+    contextSrv.hasPermission(AccessControlAction.ActionUserRolesAdd) &&
+    contextSrv.hasPermission(AccessControlAction.ActionUserRolesRemove);
+
+  useEffect(() => {
+    setOrgId(currentOrgId);
+  }, [currentOrgId, setOrgId]);
```

Comment:
Same comment here, the org ID is not likely to change during a lifecycle of this component.

Reply:
Removed, thanks!